exercises/e001/results/propiro/outputBase32faster.py

```diff
@@ -0,0 +1,32 @@
+import string
+import random
+import argparse
+
+parser = argparse.ArgumentParser(description='base32 key generator script')
+parser.add_argument('-s','--size', const=32, default=32, type=int, help='size of key, default=32', nargs='?')
+caseGroup = parser.add_mutually_exclusive_group()
+caseGroup.add_argument('-u', action="store_true", help='use uppercase letters, default option', default=True)
+caseGroup.add_argument('-l', action="store_true", help='use lowercase letters', default=False)
```

Comment:
Can you make them conflicting in `argparse` ?

Reply:
in case user try calling both -u & -l argparse will notify him about picking only one argument, preventing from using them at once, unless thats not how I understand "conflicting", then I'd need further explanation.

Reply:
Was that a question? :)

Reply:
Oh so you want me to strip the functionality of preventing it from working correctly? Sorry, misunderstood the comment,

Reply:
Ok, I see you used `add_mutually_exclusive_group()` which makes it conflicting.

Reply:
and to show this:

```
$ python3 outputBase32faster.py -l -u
usage: outputBase32faster.py [-h] [-s [SIZE]] [-u | -l]
outputBase32faster.py: error: argument -u: not allowed with argument -l
```